libs/sdk-ui-dashboard/src/presentation/dashboard/components/OverlayProvider.tsx

```diff
@@ -0,0 +1,147 @@
+// (C) 2022 GoodData Corporation
```

Comment:
This Provider should likely live next to the Overlay component in `sdk-ui-kit`, not in `sdk-ui-dashboard`

Reply:
This components has very specific layout that deepens to Dashboard structure. I will not place to UI-Kit since this component is very Dashboard specific.

Reply:
Moved to SDK-UI-KIT